modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -209,19 +209,21 @@ function MultipleChoice(props) {
       setOptions((oldOptions) => {
         let newOptions = oldOptions.slice();
         newOptions.push([name, id, false]);
-        return newOptions
+        return newOptions;
       });
     }
   }
 
   // Remove a non-default option
   let removeOption = (id, name) => {
     onChange && onChange(id); // will trigger callback in Form.jsx
-    setOptions(options.filter(
-      (option) => {
-        return !(option[VALUE_POS] === id && option[LABEL_POS] === name)
-      }
-    ));
+    setOptions( (old) => {
+      let newOptions = old.filter(
+        (option) => {
+          return !(option[VALUE_POS] === id && option[LABEL_POS] === name)
```

Comment:
This will probably prevent the removal of default options (not tested):
```suggestion
          return !(option[VALUE_POS] === id && option[LABEL_POS] === name) || option[IS_DEFAULT_POS]
```